rxros_tf/include/rxros_tf/rxros_tf.h

```diff
@@ -77,15 +77,15 @@ namespace rxros
         auto send_transform() {
             return [=](auto&& source) {
                 tf::TransformBroadcaster transformBroadcaster;
-                source.observe_on(rxcpp::synchronize_new_thread()).subscribe(
+                source.subscribe_on(rxcpp::synchronize_new_thread()).subscribe(
```

Comment:
I suggest removing observe_on here as well (not just replacing it with subscribe_on).